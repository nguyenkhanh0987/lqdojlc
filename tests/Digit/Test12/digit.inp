100
843077593 908362964
839660739 962022489
316165343 407720943
47449407 678512313
272451032 938852901
484235542 542443852
492604413 642180397
638659367 817894456
903357405 987042969
193043753 489467543
834004171 861965474
662825672 796138859
569469732 702565702
310966350 397487419
44194624 369157402
607583576 906718785
42729551 534939379
888340660 976124175
988741476 997059567
886989046 930115210
218336004 313206925
200991779 282930280
679963507 804886371
967595884 970945743
75829901 231034157
681106761 998116350
210776745 450162821
128793083 529696595
829715752 965705962
661719057 985703064
303810628 670560611
569558845 834958667
334314839 998093967
447739094 713075920
634648816 819746307
889355151 938837857
153214782 524764188
117987273 689621903
434533944 731326633
837086581 998018809
579721291 806742769
409470579 593600096
281403762 471663868
925067645 980862779
639909310 880095925
404364495 868713466
189148736 294009471
766443167 986253718
925724388 958330924
655070954 885502445
193316921 598729180
215555258 816753497
949487114 986153128
758447834 911265891
411572775 639826691
527499783 902927434
52036272 75426095
276213418 934152614
129202932 358498704
401083852 405971567
551254414 876220965
935215813 965603129
182712155 253593310
99781379 593703660
328476127 799108377
678902582 702765851
288402330 818719392
597349467 818967775
403580479 699359767
375484603 945903434
950274173 998143724
2574641 197450647
143738263 678974249
847035688 989307376
110705597 391759425
199431115 445648036
216116714 320985864
742058889 836014067
154721676 317870253
502682010 723720882
933190819 939092155
275345886 977050494
161028494 201869816
24572021 174230757
78335362 376368568
61987975 583991973
956278008 995379244
212062241 438117130
769382185 803806777
94671672 986284855
981915805 988752168
993846337 999681980
271767277 605223218
714468205 801521746
563811586 949839803
744467764 770615451
756284036 834790314
146560086 205221744
495951613 725114464
792911300 829225679
