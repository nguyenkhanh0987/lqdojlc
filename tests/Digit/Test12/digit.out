5737953
8296749
14544711
97355141
74634037
6248030
15846389
17228283
4520035
45845681
1782723
13844473
15876028
14106614
59061628
25930437
82230645
5071870
1113615
2267117
16647914
13023007
12294544
229047
29148595
24994289
36263146
63752501
9061827
26201571
45929442
25639913
69543332
31848481
17427246
2490210
59154276
83606286
34086206
11841160
22963318
23815913
28002600
3257163
21320268
47456942
18888289
16291203
1452004
20558763
59788647
75937506
3054857
12862151
26854231
36371589
5651127
73630395
39667809
368648
31159223
1703617
11590395
76456582
57143216
3678779
63177585
19724986
35266229
58382961
4367342
39209314
77904960
10103779
48909189
37064249
17396459
7941919
29701509
22797233
144911
76497273
9988464
30079267
52789866
84645650
3628560
34689845
4460674
109532084
650770
508460
47962621
8175711
33770998
2277261
6715266
13110830
23757680
1809234
